modules/tinymce/src/plugins/link/main/ts/api/Commands.ts

```diff
@@ -1,17 +1,20 @@
 import Editor from 'tinymce/core/api/Editor';
 
-import * as Actions from '../core/Actions';
+import * as Dialog from '../ui/Dialog';
 import * as Options from './Options';
 
+interface Dialog {
+  readonly 'dialog': boolean;
+}
+
 const register = (editor: Editor): void => {
-  editor.addCommand('mceLink', () => {
-    if (Options.useQuickLink(editor)) {
-      // Taken from ContextEditorEvents in silver. Find a better way.
+  editor.addCommand('mceLink', (_ui, value?: Dialog) => {
+    if (value.dialog || !Options.useQuickLink(editor)) {
```

Comment:
`value` can be `undefined` here so this might throw an error. I'd suggest we do something like this instead:
```suggestion
    if (value?.dialog === true || !Options.useQuickLink(editor)) {
```